app/broadcast_message/rest.py

```diff
@@ -49,12 +49,8 @@ def _update_broadcast_message(broadcast_message, new_status, updating_user):
         )
 
     if new_status == BroadcastStatusType.BROADCASTING:
-        # TODO: Remove this platform admin shortcut when the feature goes live
-        if updating_user == broadcast_message.created_by and not (
-            # platform admins and trial mode services can approve their own broadcasts
-            updating_user.platform_admin or
-            broadcast_message.service.restricted
-        ):
+        # trial mode services can approve their own broadcasts
```

Comment:
We call it ‘training mode’ in the interface now
```suggestion
        # training mode services can approve their own broadcasts
```